lib/psulib_traject/workers/hourly_indexer.rb

```diff
@@ -4,13 +4,13 @@ module PsulibTraject
   module Workers
     class HourlyIndexer < Base
       def perform
-        target = Dir.glob("#{ConfigSettings.symphony_data_path}/#{ConfigSettings.symphony_hourlies_subdir}/**/*.mrc")
+        target = Dir.glob("#{ConfigSettings.symphony_data_path}/#{ConfigSettings.symphony_hourlies_subdir}/**/*.m*rc")
         indexed_files = redis.keys('hr:*').map! { |e| e.gsub('hr:', '') }
```

Comment:
You're calling `map!` on the array, which means you'd be modifying the actual array, and overwriting the elements within `redis.keys`. I'm not sure that's what you want... maybe just `map` here since indexed_files is capturing the new output?

Reply:
Good catch. Thanks, StackOverflow.